quality/errorprone/android.gradle

```diff
@@ -1,10 +1,16 @@
 apply plugin: 'net.ltgt.errorprone'
 
 dependencies {
-  annotationProcessor "com.uber.nullaway:nullaway:0.4.6"
-  testAnnotationProcessor "com.uber.nullaway:nullaway:0.4.6"
+  if(JavaVersion.current().getMajorVersion().toInteger() < 9) {
+    // see https://github.com/tbroyer/gradle-errorprone-plugin#jdk-8-support
+    errorproneJavac("com.google.errorprone:javac:9+181-r4173-1")
```

Comment:
Isn't this done by the gradle plugin? I don't have it in my project and it works just fine.
I only have:

```groovy
apply plugin: 'net.ltgt.errorprone'

dependencies {
  errorprone 'com.google.errorprone:error_prone_core:2.3.2'
  'com.uber.nullaway:nullaway:0.6.0'.with {
    annotationProcessor it
    testAnnotationProcessor it
  }
}
```

Reply:
my jdk 8 failed, complaining about the missing jdk9 compiler, so i followed the README of the error prone plugin and it works as they instruct.